generators/entity-client/templates/vue/src/main/webapp/app/entities/entity-update.vue.ejs

```diff
@@ -37,10 +37,10 @@
                         <%_ if (fields[idx].fieldIsEnum) { _%>
                         <select class="form-control" name="<%= fieldName %>" :class="{'valid': !$v.<%= entityInstance %>.<%= fieldName %>.$invalid, 'invalid': $v.<%= entityInstance %>.<%= fieldName %>.$invalid }" v-model="$v.<%= entityInstance %>.<%= fieldName %>.$model" id="<%= entityFileName %>-<%= fieldName %>" <% if (fields[idx].fieldValidate === true && fields[idx].fieldValidateRules.includes('required')) { %> required<% } %>>
                         <%_ const enumPrefix = angularAppName + '.'+ fieldType;
-                            const values = fields[idx].fieldValues.replace(/\s/g, '').split(',');
+                            const values = getEnumValuesWithCustomValues(fields[idx].fieldValues);
                             for (key in values) {
-                                const value = values[key]; _%>
-                            <option value="<%= value %>" v-bind:label="$t('<%=enumPrefix%>.<%=value%>')"><%=value%></option>
+                                const enumValue = values[key]; _%>
+                            <option value="<%= enumValue.name %>" v-bind:label="$t('<%=enumPrefix%>.<%= enumValue.name %>')"><%= enumValue.name %></option>
```

Comment:
```suggestion
                            <option value="<%= enumValue.name %>" v-bind:label="$t('<%=enumPrefix%>.<%= enumValue.value %>')"><%= enumValue.value %></option>
```
Correct?

Reply:
Thanks @mshima : I'm trying this and commiting your suggestion